hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/command/CommandStatusReportHandler.java

```diff
@@ -54,32 +55,14 @@ public void onMessage(CommandStatusReportFromDatanode report,
     cmdStatusList.forEach(cmdStatus -> {
       LOGGER.trace("Emitting command status for id:{} type: {}", cmdStatus
           .getCmdId(), cmdStatus.getType());
-      switch (cmdStatus.getType()) {
-      case replicateContainerCommand:
-        publisher.fireEvent(SCMEvents.REPLICATION_STATUS, new
-            ReplicationStatus(cmdStatus));
-        if (cmdStatus.getStatus() == CommandStatus.Status.EXECUTED) {
-          publisher.fireEvent(SCMEvents.REPLICATION_COMPLETE,
-              new ReplicationManager.ReplicationCompleted(
-                  cmdStatus.getCmdId()));
-        }
-        break;
-      case deleteBlocksCommand:
+      if (cmdStatus.getType() == SCMCommandProto.Type.deleteBlocksCommand) {
```

Comment:
DeleteBlock handler is the only user of CommandWatcher now ?

Reply:
Yes, no other command uses CommandWatcher anymore.